site/realm/functions/promoteSubmissionToReport.js

```diff
@@ -121,21 +120,26 @@ exports = async (input) => {
         if (submission.user) {
           await subscriptionsCollection.insertOne({
             type: 'submission-promoted',
-            incident_id: BSON.Int32(newIncident.incident_id),
+            incident_id: BSON.Int32(parentIncident.incident_id),
```

Comment:
Would you please add tests that would catch this bug in the future?
https://github.com/responsible-ai-collaborative/aiid/blob/0c0ec744c4216e79f019e99cae936749cea1e773/site/gatsby-site/cypress/e2e/unit/functions/promoteSubmissionToReport.cy.js

(commenting again to return the code review)